velox/functions/sparksql/tests/ElementAtTest.cpp

```diff
@@ -43,10 +43,10 @@ class SubscriptTest : public SparkFunctionBaseTest {
 } // namespace
 
 // Spark's subscript ("a[1]") behavior:
```

Comment:
Let's clarify that this behavior is only when spark.sql.ansi.enabled = false.

Reply:
Thanks, fixed it.